llvm/lib/Transforms/Vectorize/VPlanRecipes.cpp

```diff
@@ -1504,49 +1504,50 @@ void VPWidenCastRecipe::execute(VPTransformState &State) {
     setFlags(CastOp);
 }
 
+// Computes the CastContextHint from a recipes that may access memory.
```

Comment:
```suggestion
/// Computes the CastContextHint for a recipe.
```

It computes the win for any recipe I think

Reply:
Updated, thanks.